components/head-static.html

```diff
@@ -27,6 +27,14 @@
 <meta ng-if="about.description" name="description" content="{{ about.description }}"/>
 <meta ng-if="about.keywords" name="keywords" content="{{ about.keywords }}"/>
 
+<!-- Web component polyfill loader -->
+<!-- Load polyfills; note that "loader" will load these async -->
+<script src="https://unpkg.com/@webcomponents/webcomponentsjs@2.0.0/webcomponents-bundle.js"></script>
```

Comment:
mind trying https://unpkg.com/@webcomponents/webcomponentsjs@2.1.3/webcomponents-loader.js so we don't pull in the polyfills when we don't need them?

Reply:
@davidmsibley Done. Seems to function the same across browsers :)